http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/MonitorConfig.scala

```diff
@@ -1,10 +1,10 @@
 package org.broadinstitute.dsde.workbench.leonardo.config
 
+import org.broadinstitute.dsde.workbench.leonardo.model.google.ClusterStatus
+
 import scala.concurrent.duration.FiniteDuration
 
 /**
   * Created by rtitle on 9/6/17.
   */
-case class MonitorConfig(pollPeriod: FiniteDuration,
-                         maxRetries: Int = -1,
-                         recreateCluster: Boolean = true)
+case class MonitorConfig(pollPeriod: FiniteDuration, maxRetries: Int = -1, recreateCluster: Boolean = true, monitorStatusTimeouts: Map[ClusterStatus, FiniteDuration])
```

Comment:
Nit: I'd love for lines to be 120 or less char long so we can split screen on IntelliJ without scrolling horizontally. I also find it easier to read when args are stacked vertically.